### ISSUE TITLE
Tray icon balloon in every closes of the mainwindow

### ISSUE THREAD

**Issue by Albvadi** [RocketChat/Rocket.Chat.Electron#1749]
## My Setup

- Operating System: Windows 10
- App Version: 3.0.0
- Installation type: MSI

- [X] I have tested with the latest version
- [X] I can simulate the issue easily

## Description
Tray icon balloon in every closes of the mainwindow

### Current Behavior
Every time I close the mainwindow, a balloon it´s showed saying me that app is´still running. I know from the first notification that the application it´s still running in background,

### Expected Behavior
I expect only one notification the first that I close the application´s mainwindow, but not all the times, same as other windows applications. If anything, the first time after every restart of the application process.

**Comment by thuguet** [comment]
I also have users complaining of that,
they would wan't to have the ability to disable this specific notification, but not the other one.

**Comment by stevenhfotofix** [comment]
Its extremely annoying, we need an option to remove this completely useless notification.

**Comment by Chealer** [comment]
This persists in 3.0.2. I recommend to pull version 3 until this is fixed.

**Comment by daztop** [comment]
Just to add to this - Skype shows a similar notification, but includes a message indicating that clicking the notification will dismiss it and not show again.  I can't imagine why anyone would think it a good idea to show it every single time.

**Comment by Cancer-zern** [comment]
Windows 10 x64, version 3.0.4, exe installation, same issue

**Comment by Dark-Passenger** [comment]
can these lines be removed

https://github.com/RocketChat/Rocket.Chat.Electron/blob/0adc73c2ecfe7010adbb06583b22c55ff172a169/src/ui/main/trayIcon.ts#L122-L124

they are the source of this popup and it's win32 only and really annoying.

alternatively maybe add a setting to enable/disable this "feature" ?

**Comment by eduardhc** [comment]
> can these lines be removed
> 
> https://github.com/RocketChat/Rocket.Chat.Electron/blob/0adc73c2ecfe7010adbb06583b22c55ff172a169/src/ui/main/trayIcon.ts#L122-L124
> 
> they are the source of this popup and it's win32 only and really annoying.
> 
> alternatively maybe add a setting to enable/disable this "feature" ?

More than removing, I'd say better just add a flag so it is only shown the first time the window is closed to tray. And serializing it somewhere, so it doesn't que get shown again on next app executions.

**Comment by Dark-Passenger** [comment]
> > can these lines be removed
> > https://github.com/RocketChat/Rocket.Chat.Electron/blob/0adc73c2ecfe7010adbb06583b22c55ff172a169/src/ui/main/trayIcon.ts#L122-L124
> > 
> > they are the source of this popup and it's win32 only and really annoying.
> > alternatively maybe add a setting to enable/disable this "feature" ?
> 
> More than removing, I'd say better just add a flag so it is only shown the first time the window is closed to tray. And serializing it somewhere, so it doesn't que get shown again on next app executions.

that would be great but if that's not something that will be built by the next release, I would suggest just disabling this and enabling it once we actually have a setting or option to disable it.

**Comment by stevenhfotofix** [comment]
> > > can these lines be removed
> > > https://github.com/RocketChat/Rocket.Chat.Electron/blob/0adc73c2ecfe7010adbb06583b22c55ff172a169/src/ui/main/trayIcon.ts#L122-L124
> > > 
> > > they are the source of this popup and it's win32 only and really annoying.
> > > alternatively maybe add a setting to enable/disable this "feature" ?
> > 
> > 
> > More than removing, I'd say better just add a flag so it is only shown the first time the window is closed to tray. And serializing it somewhere, so it doesn't que get shown again on next app executions.
> 
> that would be great but if that's not something that will be built by the next release, I would suggest just disabling this and enabling it once we actually have a setting or option to disable it.

I have stopped my users from updating the client, until this is resolved.  Yes, it is that big an issue for our team.   I second @Dark-Passenger in just commenting it out until there is an official option (or not, our team used it perfectly well for years without being told that the window was still open - Heck, OSX does that by default for applications all the time, keeping it open, even when the main window is closed, without the user having to be explicitly told so.)

**Comment by RavuAlHemio** [comment]
Showing the message only once per execution is an improvement, but still a tad too annoying. Please improve this further, either by showing the message only once (and persisting the flag that stores that the user has seen this message before), as recommended by @eduardhc, or by providing a user preference to not display these messages at all, as recommended by @Dark-Passenger.

Thanks!

**Comment by combin** [comment]
Almost a year not activity in this. I need create new issue for improving this? See this popup every day is rally annoying!

**Comment by combin** [comment]
Made new issue https://github.com/RocketChat/Rocket.Chat.Electron/issues/2188
